### PR TITLE
fix[ATLAS-154]: update country code data for phone number and currency module

### PR DESCRIPTION
**Pull request by RgnDunes**
# Description

Currently the country code mappings we have in different modules is not in sync i.e., for phone number/currency module we have data for say 190 country codes but same for currencies may be 200 etc

## Changes Made

List the main changes made in this pull request.

| Title                  | --- |
| ---------------------- | --- |
| JIRA link              | https://razorpay.atlassian.net/browse/ATLAS-154  |
| Slack thread (if any)  | NA  |
| Product spec           | NA  |
| Tech spec/One-pager    | NA  |
| Bundle Size Difference | NA  |
| Fixes Issue            | NA  |

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This change is a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Is your change backward compatible ?
- [ ] Tested on major browsers (Chrome, Firefox, Safari, IE) ?
- [ ] Tested in a consumer application(s) ?

## Additional Notes

Any additional information that would be helpful for the reviewer.

## Checklist:

- [x] Add Jira ID(s) in PR title and in the description?
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My code is written in TypeScript and follows the coding standards of the project.
- [x] I have added relevant documentation and updated the README if necessary.
- [x] My commit messages are clear and follow the project's commit message conventions.
- [ ] Is any external library added?
- [x] My changes do not introduce any new runtime errors or warnings.
- [x] All relevant unit tests have been added/updated in my PR for the change.
- [ ] Any Screenshots (mobile & desktop) required for PR? If yes, have you added the respective screenshots ?
- [ ] Any manual dev testing done by you on beta/func environment?
- [ ] Reviewer added (SLA 2 days)

## Reviewer Checklist

- [ ] Sufficient QA/Dev-Testing is done with proof (test cases list)
- [ ] Ensure that the change in bundle size falls within the acceptable range.

---

## PR Title Format

Format: `<type>: <subject>`

Types can be as follows:

- `feat`: (new feature for the user, not a new feature for build script)
- `fix`: (bug fix for the user, not a fix to a build script)
- `docs`: (changes to the documentation)
- `style`: (formatting, missing semi colons, etc; no production code change)
- `refactor`: (refactoring production code, eg. renaming a variable)
- `test`: (adding missing tests, refactoring tests; no production code change)
- `chore`: (updating grunt tasks etc; no production code change)

### CODE REVIEW

File: packages/i18nify-js/src/modules/types/geo.ts [hunk 1]
```diff
@@ -1,248 +1,230 @@
+// extracted from https://www.henleyglobal.com/passport-index
```
Comment (tarun-khanna) [inline packages/i18nify-js/src/modules/types/geo.ts:1]: Can you point to the exact source ? This opens up their homepage.

File: packages/i18nify-js/src/modules/types/geo.ts [hunk 2]
```diff
@@ -1,248 +1,230 @@
...
-  | 'MN'
-  | 'NP'
-  | 'TJ'
-  | 'TM'
-  | 'AZ'
-  | 'GE'
-  | 'KG'
-  | 'UZ'
-  | 'EU';
+  | 'AF' // Afghanistan
```
Comment (tarun-khanna) [inline packages/i18nify-js/src/modules/types/geo.ts:4]: How did the countryCodes reduce ?

File: packages/i18nify-js/src/modules/phoneNumber/data/phoneFormatterMapper.ts [hunk 3]
```diff
@@ -1,251 +1,231 @@
...
-  BG: 'xx xxx xxxx',
-  EE: 'xxx xxxx',
-  BY: 'xx xxx-xx-xx',
-  BA: 'xx xxx-xxxx',
-  FK: 'xxxxx',
-  BL: 'xxx xx xx xx',
-  EC: 'xx xxx xxxx',
-  CW: 'xxx xxxx',
-  BQ: 'xxx xxxx',
+  IN: 'xxxx xxxxxx', // India, Example: 7394926646 formatted to 7394 926646
```
Comment (tarun-khanna) [inline packages/i18nify-js/src/modules/phoneNumber/data/phoneFormatterMapper.ts:4]: What is the reason for country codes getting reduced in new changes ?

File: packages/i18nify-js/src/modules/types/geo.ts (hunk 2 above)
Comment (RgnDunes) [reply packages/i18nify-js/src/modules/types/geo.ts:4]: A couple of them were uninhabited islands which are not even recognized by Passport Index so removed those plus some were merged into one country so modified those as well.

File: packages/i18nify-js/src/modules/phoneNumber/data/phoneFormatterMapper.ts (hunk 3 above)
Comment (RgnDunes) [reply packages/i18nify-js/src/modules/phoneNumber/data/phoneFormatterMapper.ts:4]: CC: Cocos (Keeling) Islands take this for example, this has been removed it's a remote territory of Australia in the Indian Ocean. Similarly there were a bunch of country codes which were either uninhabited or were not recognised or were merged into a single country. So removing those.

**Review by tarun-khanna — APPROVED** [review]

**Pull request merged**